vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -55,21 +56,18 @@ testImagesPulled() {
     return
   fi
 
-  imagesNotPulled=()
+  imagesToBePulled=$(echo $imagesToBePulled | jq ".ContainerImages" | jq .[] --monochrome-output --compact-output)
```

Comment:
nit: `echo $imagesToBePull` == `cat $COMPONENTS_FILEPATH`?
the variable here is shadowing line 48